google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -158,8 +158,7 @@ class TracingMessageBatch : public MessageBatch {
       message_spans.swap(message_spans_);
     }
 
-    auto batch_sink_spans =
-        MakeBatchSinkSpans(std::move(message_spans), options_);
+    auto batch_sink_spans = MakeBatchSinkSpans(message_spans, options_);
```

Comment:
I think this copy is unnecessary and we should change the signature to `MakeBatchSinkSpans(Spans const&, ...)`